src/app/containers/FrontPageMain/index.jsx

```diff
@@ -2,15 +2,49 @@
 import React, { useContext } from 'react';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import path from 'ramda/src/path';
+import Grid, { FrontPageGrid } from '#app/components/Grid';
 import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
-import { Grid, GridItemConstrainedLargeWithTopMargin } from '#lib/styledGrid';
 import { ServiceContext } from '#contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import MetadataContainer from '../Metadata';
 import LinkedData from '../LinkedData';
 import ATIAnalytics from '../ATIAnalytics';
 import ChartbeatAnalytics from '../ChartbeatAnalytics';
 
+const mainColumns = {
```

Comment:
We could have all this configuration in a separate file, where we could also set how the other containers behave in the future, and then just import here the specific one for Front Pages.

Reply:
Good idea, atm we'll have this on the `frontPageMain` file just to be clear on how these values are set. In future, we should move this to a separate config file that will hold all column config for various pages and components.

Reply:
If you prefer to do it later on, I am happy with it 👍